modules/standard/FileSystem.chpl

```diff
@@ -1463,12 +1587,17 @@ proc rmTree(out error: syserr, root: string) {
 
    :throws SystemError: Thrown to describe an error if one occurs.
 */
-proc sameFile(file1: string, file2: string): bool throws {
+//ENGIN: Instead of adding bunch of overloads, I made this generic.
+proc sameFile(file1, file2): bool throws {
```

Comment:
how about a where clause or compilerError if file1/ file2 are not bytes or string